monitor.go

```diff
@@ -11,9 +11,10 @@ type JobStatus string
 
 // The different statuses of job that can be used.
 const (
-	Fail    JobStatus = "fail"
-	Success JobStatus = "success"
-	Skip    JobStatus = "skip"
+	Fail        JobStatus = "fail"
+	Success     JobStatus = "success"
+	Skip        JobStatus = "skip"
+	Rescheduled JobStatus = "rescheduled"
```

Comment:
might be more helpful to make this `singleton_rescheduled`? I could see it being unclear if this is just a regular job being rescheduled or something rescheduled due to a limit being reached.

Reply:
Hi I've changed this to `singleton_rescheduled`. You could review again now.